examples/multi-language/src/main/java/org/apache/beam/examples/multilanguage/PythonDataframeWordCount.java

```diff
@@ -42,19 +41,20 @@
  * An example that counts words in Shakespeare and utilizes a Python external transform.
  *
  * <p>This class, {@link PythonDataframeWordCount}, uses Python DataframeTransform to count words
- * from the input text file. The Python expansion service provided by --expansionService must allow
- * the expansion of apache_beam.dataframe.transforms.DataframeTransform (which can be done by
- * passing --fully_qualified_name_glob commandline option when launching the expansion service).
+ * from the input text file.
  *
- * <p>Note that, for using Dataflow Runner, you should specify the following two additional
- * arguments:
+ * <p>The example command below shows how to run this pipeline on Dataflow runner:
  *
  * <pre>{@code
- * --experiments=use_runner_v2
- * --sdkHarnessContainerImageOverrides=.*python.*,gcr.io/apache-beam-testing/beam-sdk/beam_python3.8_sdk:latest
+ * ./gradlew :examples:multi-language:pythonDataframeWordCount --args=" \
+ * --runner=DataflowRunner \
+ * --output=gs://{$OUTPUT_BUCKET}/count \
+ * --experiments=use_runner_v2 \
```

Comment:
Is this required ? For Python if we detect multi-language transforms for DataflowRunner we add this experiment automatically (can be updated in a separate PR).

Reply:
I couldn't submit the job without `use_runner_v2` flag. We can look into it in a separate PR.